src/wp-includes/l10n/class-wp-translation-controller.php

```diff
@@ -434,4 +434,20 @@ protected function get_files( string $textdomain = 'default', ?string $locale =
 
 		return $this->loaded_translations[ $locale ][ $textdomain ] ?? array();
 	}
+
+	/**
+	 * Returns a boolean to indicate whether a translation exists for a given string with optional text domain and locale.
+	 *
+	 * @param string $singular  Singular translation to check.
+	 * @param string $textdomain  Optional. Text domain. Default 'default'.
+	 * @param ?string $locale  Optional. Locale. Default current locale.
```

Comment:
As per the [coding standards](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/), these should all be aligned.

Let's also tentatively add `@since 6.7.0`

Reply:
Done